setup.py

```diff
@@ -58,12 +58,12 @@ def prerelease_local_scheme(version):
         'Cython>=0.25.2',
         'scikit-build>=0.8.1',
         'cmake>=0.6.0',
+        'numpy>=1.12.1',
     ],
     install_requires=[
         'ctk-cli>=1.5',
         # scientific packages
         'nimfa>=1.3.2',
-        'numpy>=1.12.1',
```

Comment:
numpy needs to remain in the `install_requires` section (in addition to the `setup_requires` section).  The `setup_requires` section is what is necessary before you can install; the `install_requires` section is needed to run it.  If you install from a wheel, you don't have to install the `setup_requires` components, and therefore (I think), installing from a wheel wouldn't install numpy.

Reply:
Got it, thanks! OK done.